src/pages/AppList.svelte

```diff
@@ -1,24 +1,26 @@
 <script>
+  import { onMount } from "svelte";
   import FilterInput from "../components/FilterInput.svelte";
+  import { fetchJSON } from "../state/api";
 
   const URL = "data/apps.json";
   let apps;
   let filteredApps;
-  fetch(URL)
-    .then((r) => r.json())
-    .then((ret) => {
-      apps = ret.sort((a, b) => a.app_id > b.app_id);
-      filteredApps = apps;
-    });
+
+  onMount(async () => {
+    apps = await fetchJSON(URL);
+    apps = apps.sort((a, b) => a.app_id > b.app_id);
```

Comment:
Just a nit, but it's generally nicer to not have to reassign to the same variable. Maybe something along the lines of `let data = ...; apps = data.sort(...)`.

Reply:
Thanks, It doesn't need to be re-assigned as the sort() function is in-place algorithm.

Reply:
I see, I was thinking `sorted` like python. This would be error, then?

Reply:
Yes, I added it on a new line as ```apps.sort(...)``` which works fine.